toolset/setup/linux/languages/elixir.sh

```diff
@@ -1,16 +1,13 @@
 #!/bin/bash
 
-# TODO seems to be broken installation
-echo "WARN: Elixir does not install"
-return 1
+RETCODE=$(fw_exists ${IROOT}/elixir.installed)
+[ ! "$RETCODE" == 0 ] || { return 0; }
 
-# fw_exists v0.13.3.tar.gz
-# [ $? -ne 0 ] || { return 0; }
+wget http://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb
```

Comment:
Then why is it here? Specify that `elixir` requires `erlang` via `fw_depends erlang` in an early line.